platform/pulpcore/app/apps.py

```diff
@@ -17,6 +17,23 @@ def pulp_plugin_configs():
         if isinstance(app_config, PulpPluginAppConfig):
             yield app_config
 
+def pulp_plugin_config(plugin_app_label):
+    """
+    A getter of specific pulp plugin config
+
+    This makes it easy to retrieve a config for a specific Pulp plugin when looking for a
+    registered model.
+
+    Args:
+        plugin_label (basestring): Django app label of the pulp plugin
+
+    Returns:
```

Comment:
This needs a `Raises` section in the google style docstring for the RuntimeError